laze-project.yml

```diff
@@ -266,6 +266,16 @@ contexts:
     env:
       PROBE_RS_CHIP: nrf5340_xxAA
 
+  - name: nrf5340-net
+    parent: nrf53
+    provides:
+      - has_storage_support
+    selects:
+      - cortex-m33
```

Comment:
Just a minor thing concerning the order: in the rest of the file we have the `selects` section before the `provides` section, it would be nice to have the same ordering here.